specification/ingest/delete_ip_location_database/DeleteIpLocationDatabaseRequest.ts

```diff
@@ -22,26 +22,31 @@ import { Ids } from '@_types/common'
 import { Duration } from '@_types/Time'
 
 /**
- * Deletes an IP location database configuration.
+ * Delete IP geolocation database configurations.
  * @rest_spec_name ingest.delete_ip_location_database
  * @availability stack since=8.15.0 stability=stable
  * @availability serverless visibility=private
+ * @cluster_privileges manage
  */
 export interface Request extends RequestBase {
   path_parts: {
     /**
-     * A comma-separated list of IP location database configurations to delete
+     * A comma-separated list of IP location database configurations.
      */
     id: Ids
   }
   query_parameters: {
     /**
-     * Period to wait for a connection to the master node.
+     * The period to wait for a connection to the master node.
      * If no response is received before the timeout expires, the request fails and returns an error.
-     * @server_default 30s */
+     * A value of `-1` indicates that the request should never time out.
+     * @server_default 30s
+     */
     master_timeout?: Duration
     /**
-     * Period to wait for a response. If no response is received before the timeout expires, the request fails and returns an error.
+     * Period to wait for a response.
```

Comment:
for parallelism w the other edit

```suggestion
     * The period to wait for a response.
```